### ISSUE TITLE
Xcode build fails with 'UnityAds/UADSBanner.h' file not found

### ISSUE THREAD

**Issue by z4gon** [getsentry/sentry-unity#463]
### Environment

How do you use Sentry?
Sentry SaaS

Which SDK and version?
sentry-unity v0.8.0

### Steps to Reproduce

1. Create a core 2D Unity project
2. Add package UnityAds v4.0.0 (also v3.7.5 will repro the issue)
3. (optional) Add package UnityAds iOS 14 Support v1.2.0
4. Add package Unity-Sentry v0.8.0 via git url

### Expected Result

Project will build ok in xcode.
Project will build ok in unity cloud build for ios (using xcode).

### Actual Result

Xcode build fails.
Unity cloud build fails for ios.

Error message:
```bash
/Libraries/com.unity.ads/Plugins/iOS/UnityBannerUnityWrapper.m:1:9: 'UnityAds/UADSBanner.h' file not found
```

![build-error](https://user-images.githubusercontent.com/4588601/145620262-b6abd641-4838-44c2-a781-993b239cba30.png)

**Comment by CodinRonin** [comment]
Check the build steps. I've had a similar issue with integrating sentry and another third party sdk was having the same error - headers missing.
The culprit turned out to be that the ios postprocessing step that sentry does overrides the FRAMEWORK_SEARCH_PATH param that was already set by the other framework. 

[sentry ios postprocess](https://github.com/getsentry/sentry-unity/blob/main/src/Sentry.Unity.Editor.iOS/BuildPostProcess.cs)
[sentryXcodeProject.AddSentryFramework()](https://github.com/getsentry/sentry-unity/blob/913b665403cf8bfbdb98ab74901300f94e166e71/src/Sentry.Unity.Editor.iOS/SentryXcodeProject.cs#L59)

**Comment by bitsandfoxes** [comment]
Thanks for bringing this to our attention. And thanks @CodinRonin for sharing this bit of information. There might be an easy solution.
I'll take a look and get back to you.

**Comment by bitsandfoxes** [comment]
Turns out you were completely right, @CodinRonin. We were overwriting the build property here: https://github.com/getsentry/sentry-unity/blob/913b665403cf8bfbdb98ab74901300f94e166e71/src/Sentry.Unity.Editor.iOS/SentryXcodeProject.cs#L72
#466 aims to fix that and 0.8.1 should include it.

**Comment by z4gon** [comment]
Thank you so much!

**Comment by bruno-garcia** [comment]
#466 is getting published in 0.9.0 coming out tomorrow
